src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -59,136 +61,91 @@ const defaultProps = {
 // We also update the state on layout changes which will be triggered often.
 // There will be n number of tooltip components in the page.
 // It's good to memorize this one.
```

Comment:
I know you didn't write this line, but while we're in here, can we update it please:
```suggestion
// It's good to memoize this one.
```

Reply:
Done 👍